src/quiz.cn.json

```diff
@@ -0,0 +1,1040 @@
+    "show": "2",
+    "category": "licensing",
+    "answers": [
+      {
+        "answer": "可以，我们可以分发任何基于目录X的软件。 Yes for any Category X bundled software.",
+        "correct": false,
+        "level": 1,
+        "hint": "禁止的协议的软件是不能分发的。 Prohibited licensed software can not be distributed."
+      },
+      {
+        "answer": "不可以（但是有一些特殊的意外情况）No (with a couple of rare exceptions).",
+        "correct": true,
+        "level": 1,
+        "hint": "禁止的协议的软件是不能分发的。 Prohibited licensed software can not be distributed."
+      },
+      {
+        "answer": "可以，但是只能针对只能针对一些通用的软件构建工具。 Yes but only for some common build tools.",
+        "correct": true,
+        "level": 3,
+        "hint": "有一些禁止许可协议的构建工具软件是能够分发的。 Some prohibited licensed build tool software can be distributed."
+      }
+    ],
+    "reason": "禁止的协议的软件是不能分发的。 Prohibited licensed software can not be distributed.",
+    "links": [
+      {
+        "name": "禁止分发",
+        "link": "https://www.apache.org/legal/resolved.html#prohibited"
+      },
+      {
+        "name": "构建工具",
+        "link": "https://www.apache.org/legal/resolved.html#build-tools"
+      }
+    ]
+  },
+  {
+    "question": "Apache项目可以依赖使用禁止使用（目录X）协议的软件吗？Can Apache rely on products under a prohibited ('Category X') license?",
+    "show": "2",
+    "category": "licensing",
+    "answers": [
+      {
+        "answer": "对于所有的依赖都是可以的。 Yes for all dependencies.",
+        "correct": false,
+        "level": 1,
+        "hint": "Apache 项目是不能依赖禁止使用的协议的软件的。 Prohibited licensed software can not be an external dependency."
+      },
+      {
+        "answer": "不行 No.",
+        "correct": true,
+        "level": 1,
+        "hint": "Apache项目是不能依赖禁止使用的协议的软件的。 Prohibited licensed software can not be an external dependency."
+      },
+      {
+        "answer": "只有一些特殊的可选依赖可以。 Only if it's an optional dependency.",
+        "correct": true,
+        "level": 2,
+        "hint": "是不是大部分的用户需要使用这些可选的依赖组件。 Do the majority of the users need this optional component?"
+      }
+    ],
+    "reason": "大部分的禁止协议软件是不能分发的，除非你依赖了一下特殊的可选依赖。 Prohibited licensed software can not be distributed but you can rely on optional dependencies.",
+    "links": [
+      {
+        "name": "Optional dependency",
+        "link": "https://www.apache.org/legal/resolved.html#optional"
+      }
+    ]
+  },
+  {
+    "question": "如果第三方的绑定依赖软件使用了双协议，你需要怎么做？ If a third party bundled bit of software is dual licensed what do you do?",
+    "show": "4",
+    "category": "licensing",
+    "answers": [
+      {
+        "answer": "将这些协议都包含至 LICENSE中。 Include both licenses in LICENSE.",
+        "correct": false,
+        "level": 2,
+        "hint": "选择一个最自由的协议，并使用它。 Select the most permissive license and use that."
+      },
+      {
+        "answer": "通过扔硬币的方式来选择一个协议包含在 LICENSE 中。 Toss a coin and put one license in LICENSE.",
+        "correct": true,
```

Comment:
@justinmclean  Are you sure this answer is true?

Reply:
You just need to pick one by random.